samples/compute/README.md

```diff
@@ -0,0 +1,204 @@
+you have authorized the sample correctly.
+
+For a full list of available client libraries, including other Google client libraries and 
+third-party open source libraries, see the [client libraries page](https://cloud.google.com/compute/docs/api/libraries).
+
+To view the full code example with all of the necessary imports, see the [create_instance.py file](create_instance.py).
+
+## Objectives
+
+ * Perform OAuth 2.0 authorization using the oauth2client library
+ * Create an instance using the google-python-client library
+ * List instances using the google-python-client library
+ * Delete an instance using the google-python-client library
+
+## Costs
+
+This tutorial uses billable components of Google Cloud including Compute Engine.
+
+## Before you begin
+
+1. In the Google Cloud Console, on the project selector page, select or create a Google Cloud project.
+   [Got to project inspector](https://console.cloud.google.com/projectselector2/home/dashboard).
+1. Make sure that billing is enabled for your Cloud project. 
+   [Learn how to confirm that billing is enabled for your project.](https://cloud.google.com/billing/docs/how-to/modify-project)
+1. [Install Cloud SDK](https://cloud.google.com/sdk)
+1. After the SDK is installed, run `gcloud auth application-default login`.
+1. Install the [google-api-python-client](http://github.com/google/google-api-python-client) library. Typically, you can run:
+   
+   ```bash
+   pip install --upgrade google-api-python-client
+   ```
+
+   You also need to have Python 2.7 or 3.3+ to run the Cloud Client Libraries for Python.
+1. Enable the Cloud Storage API.
+   ```bash
+   gcloud services enable pubsub.googleapis.com
+   ```
+1. [Create a Cloud Storage bucket](https://cloud.google.com/storage/docs/creating-buckets) and note the bucket name for later.
+
+## Authorizing requests
+
+This sample uses OAuth 2.0 authorization. There are many ways to authorize requests using OAuth 2.0,
+but for the example use [application default credentials](https://developers.google.com/accounts/docs/application-default-credentials). This lets you reuse the credentials from 
+the `gcloud` tool if you are running the sample on a local workstation or reuse credentials from a 
+service account if you are running the sample from within Compute Engine or App Engine. You should 
+have installed and authorized the `gcloud` tool in the "Before you begin" section.
+
+Application default credentials are provided in Google API Client Libraries automatically. 
+You just have to build and initialize the API:
+
+```python
+compute = googleapiclient.discovery.build('compute', 'v1')
+```
+
+See the `main()` method in the [create_instance.py](create_instance.py) script, to see how an API
+client is built and used.
+
+## Listing instances
+
+Using the Cloud Client Libraries for Python, you can list instances by using the 
+`compute.instances().list` method. 
```

Comment:
```suggestion
Using `google-api-python-client`, you can list instances by using the `compute.instances().list()` method. 
```
This line mentioned [Cloud Client Libraries](https://cloud.google.com/apis/docs/client-libraries-explained#cloud-client-libraries) but the REST based library falls under Google API Client Libraries. To avoid confusion, we should use `google-api-python-client` instead.